### DISCUSSION TITLE
Want to add AffinityScoreDefinition as a metadatatype

### DISCUSSION THREAD

**Discussion by rheacherian814** [forcedotcom/cli#2796] [Ideas]
I have followed the documentation to add this type, but when trying to make a PR, it complains that: 
unable to access 'https://github.com/forcedotcom/source-deploy-retrieve/ for my github login. Is there anyone who can help me resolve this please?

**Comment by mshanemc** [comment]
Cool, thanks!  Fork SDR, then do your PR.

**Comment by rheacherian814** [comment]
Done!
